libarchive/archive_entry.h

```diff
@@ -58,12 +58,15 @@
 #define __LA_INT64_T_DEFINED
 # if defined(_WIN32) && !defined(__CYGWIN__) && !defined(__WATCOMC__)
 typedef __int64 la_int64_t;
+typedef unsigned __int64 la_uint64_t;
```

Comment:
Duplicate the `la_uint64_t` definitions in both `archive.h` and `archive_entry.h` so that it's always available.